### PR TITLE
[FIX] mrp_analytic_cost: the module has to be depend on account_analytic_wip

### PR DESCRIPTION
**Pull request by tafaRU**
Otherwise, during its installation it raises the following error:
```
2021-08-03 11:41:06,662 169 INFO openerp_test odoo.addons.base.models.ir_ui_view: Unknown field "product.product.is_cost_type" in field analytic_product_id default domain"[('is_cost_type', '=', True)]"

View name: mrp.workcenter.form.custom
Error context:
view: ir.ui.view(1319,)
xmlid: mrp_workcenter_form_view_custom
view.model: mrp.workcenter
view.parent: ir.ui.view(953,)
file: /home/odoo/build/OCA/manufacture/setup/mrp_analytic_cost/odoo/addons/mrp_analytic_cost/views/mrp_workcenter_view.xml
```

Indeed, `is_cost_type` field is added by [account_analytic_wip](https://github.com/OCA/account-analytic/blob/d998d6e59a5a61788e5fddbdbd90a6652470ad3a/account_analytic_wip/models/product_template.py#L11).

### CODE REVIEW

**Comment by tafaRU** [comment]
@OCA/manufacturing-maintainers could you please have a look?
Note that Travis builds are not triggered  (contrary what happens in https://github.com/OCA/manufacture/pull/663).

**Comment by dreispt** [comment]
@tafaRU Thank for looking at this. I'm doing some revisions to these PRs, and expect to make some PRs soon.

**Comment by tafaRU** [comment]
@dreispt thanks for your feedback! Could you please review it?

**Comment by dreispt** [comment]
@tafaRU Hello, I'm in the process of deprecating this module.

My plan is to add two modules:
- `mrp_account_analytic`: generated "Analytic Items" on material and operation consumption. Depends on `mrp_analytic`.
- `mrp_account_analytic_wip`: extends the `mrp_account_analytic` module, to add material and operation consumption tracking, and WIP + Variance computation and posting. This also depends on `account_analytic_wip` and indirectly on `analytic_activity_based_cost`.

Thanks

**Comment by tafaRU** [comment]
> @tafaRU Hello, I'm in the process of deprecating this module.

Hello @dreispt! Out of curiosity: why we don't merge this waiting for the module will be definitely deprecated?
From my point of view this error is locking https://github.com/OCA/manufacture/pull/663 What do you think about it?

**Comment by dreispt** [comment]
OK, let's go then.

**Review by dreispt — APPROVED** [review]

**Pull request merged**
